### ISSUE TITLE
conan install -u traceback

### ISSUE THREAD

**Issue by Blubbz0r** [conan-io/conan#390]
When running ```conan install -u``` while no previous ```conan install``` was run (so e.g. no conan conanbuildinfo.cmake exists) I get the following traceback:

![image](https://cloud.githubusercontent.com/assets/11735226/17244088/59567f7c-5580-11e6-8a6f-4b026be70816.png)

**Comment by lasote** [comment]
I can't reproduce the error. Do you have a conanfile.py? conanfile.txt? Windows right?

**Comment by Blubbz0r** [comment]
It's Windows, yes. Conan version is 0.10.1. I have a simple repo with a simple conanfile.txt like this:

```
[requires]
googlemock/1.7.0@image/stable
Qt/5.7.0@image/stable

[generators]
cmake

[imports]
bin, *.dll -> ./bin # Copies all dll files from the package "bin" folder to my project "bin" folder
```

When I create a new folder and run ```conan install -u {SOURCE_DIR}``` I always get the shown traceback.

**Comment by Blubbz0r** [comment]
Actually, the Qt / short_paths.conf might be the culprit. I have tried with a different repository that does not have a Qt dependency (and thus, does not use short_paths.conf) and it is working properly.

**Comment by lasote** [comment]
Have you modified the short_paths.conf? Could you paste here?

**Comment by Blubbz0r** [comment]
There is one entry in my short_paths.conf for the Qt package:

```Qt/5.7.0@image/stable: C:/Qt/5.7.0/image/stable```

**Comment by lasote** [comment]
I get it. I've opened a PR #391 solving it, It will be fixed in next release

**Comment by memsharded** [comment]
I am also preparing a new (hopefully better) solution for the short paths limit, #392

**Comment by Blubbz0r** [comment]
Nice to hear. Let me know in case I can help with testing.

**Comment by memsharded** [comment]
This has been released in 0.11. Please upgrade and re-open if necessary. Thanks!
